### PR TITLE
Update nexus schema and graphql

### PR DESCRIPTION
**Pull request by adam2k**
## Description

Need to update the schema after the `@nexus/schema` package upgrade.

## 🎟 Issue(s)

Related to https://github.com/astronomer/houston-api/pull/528

## 🧪 Functional Testing

Test coverage and Houston API should function normally (these changes were tested in the snyk PR and the schema changes in this PR are all generated by `@nexus/schema`).

## 📋 Checklist

- [x] Rebased from the main (or release if patching) branch (before testing)
- [x] Ran `yarn lint && yarn test` before taking out of draft
- [ ] Added/updated applicable tests
- [x] Tested against [Astro-UI](https://github.com/astronomer/astro-ui/) and [Astro-CLI](https://github.com/astronomer/astro-cli/) clients.
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: src/generated/schema.graphql [hunk 1]
```diff
@@ -46,12 +46,12 @@ type AppConfig {
 }
 
 type AstroUnit {
-  actualConns: Float!
```
Comment (andriisoldatenko) [inline src/generated/schema.graphql:49]: @adam2k why do we need this changes?

Comment (andriisoldatenko) [reply src/generated/schema.graphql:49]: it make sense to add bump lib here, because you just generate new schema without any other changes.

Comment (adam2k) [reply src/generated/schema.graphql:49]: The lib changes came through a `snyk` bot change: https://github.com/astronomer/houston-api/pull/528

Should we revert that change and add these schema changes to it?

Comment (adam2k) [reply src/generated/schema.graphql:49]: I think the git history will still make sense if this is the next change that goes in.

**Review by andriisoldatenko — APPROVED** [review]

**Pull request merged**